app/templates/gentelella/users/events/_table.html

```diff
@@ -1,6 +1,23 @@
 {% macro events_table(events) %}
-    <div class="table-responsive">
-        <table class="table table-striped">
+
+<script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
```

Comment:
Why are you including the script when it's already included in this page? 

And also... In future when you do have to include scripts, don't add them like this in the middle of ye the body.